src/Pelagos/Bundle/AppBundle/Controller/UI/GomriReportController.php

```diff
@@ -37,6 +37,24 @@ public function defaultAction()
             return $this->render('PelagosAppBundle:template:AdminOnly.html.twig');
         }
           // Add header to CSV.
+        return $this->writeCsvResponse(
+            $this->getData('v1')
+        );
+    }
+
+    /**
+     * This is the version 2 Gomri Report.
+     *
+     * @Route("/v2")
+     *
+     * @return Response A Response instance
+     */
+    public function versionTwoReportAction()
+    {
+        if (!$this->isGranted('ROLE_DATA_REPOSITORY_MANAGER')) {
+            return $this->render('PelagosAppBundle:template:AdminOnly.html.twig');
+        }
+        // Add header to CSV.
         return $this->writeCsvResponse(
             $this->getData()
```

Comment:
You may want to consider setting custom filename for these two reports to avoid future confusion when downloading and comparing two versions. The custom filename is the second param of writeCsvResponse (see Report Controller)